src/payform.coffee

```diff
@@ -432,7 +432,7 @@
       prefix = prefix.toString()[0..1]
       year   = prefix + year
 
-    month = parseInt(month, 10)
+    month = parseInt(month.replace(/[\u200e]/g, ""), 10);
```

Comment:
Can you please add a comment here to explain what it is we are removing and why?